<launch>
    <node pkg="camera" type="publish_image.py" name="CameraImagePublisher" output="screen">
    </node> 
    <!-- Subscribe Topics -->
    <!-- -->
    <!-- Publish Topics -->
    <!-- "~RawRGB": sensor_msgs/Image. RGB image read from camera -->
    <!-- "~RawDepth": sensor_msgs/Image. Depth image read from camera -->
   
    <node pkg="camera" type="image_detect.py" name="ImageDetector" output="screen">

     </node> 
    <!-- Subscribe Topics -->
    <!-- "DetectImage": sensor_msgs/Image. Image from pre_process to be detected -->
    <!-- Publish Topics -->
    <!-- "~DetectResult": std_msgs/Float32MultiArray. Detect result -->

</launch>
